extensions/GitExtension/FileExplorerGitIntegration/FileExplorerGitIntegration.csproj

```diff
@@ -17,7 +17,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="LibGit2Sharp" Version="0.29.0" />
+      <PackageReference Include="LibGit2Sharp" Version="0.30.0" />

```

Comment:
Why is this version changing?

Reply:
There are some improvements in this latest version, but probably the most important one is that it updates LibGit2 to 1.7.2, which includes a couple security fixes and is recommended by the LibGit2 maintainers. https://github.com/libgit2/libgit2/releases/tag/v1.7.2